Tridentified V2.4 Power Inlet - Side mounted power inlet like Trident

Amarpal89 opened the pull request:
<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more 
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?
Tridentified V2.4 Skirts : A mod to backport the more convenient side mounting for the power inlet to the V2.4. These skirts are based on edwardyeeks "z-drive tensioner mod" compatible skirts and have power inlet and switch holes of equal geometry to the standard voron parts. 

#### How was it tested? 
The filtered inlet versions have been printed and installed by a few voron community members. Not all versions have been printed and tested but as they are based on existing voronusers repo parts with fairly straightforward modification to permit power inlet mounting I cannot foresee any issues with print-ability/fitment.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### Further notes

As these skirts have the prerequisite of requiring the user to have @edwardyeeks  z-drive tensioner mod, he and I agreed that it would be best to have them located within edwardyeeks folder.

edwardyeeks reviewed (commented):
Overall, this looks good 👍 I would sign off with your name @Amarpal89 in the readme just so that everyone is aware that this derivative of the mod is done by you.

cbc02009 reviewed (changes requested):
Since you made these changes and not edwardyeeks this should be filed under your username and not theirs.

Amarpal89 commented:
> Since you made these changes and not edwardyeeks this should be filed under your username and not theirs.

By that do you mean just adding a new row with my username in the root README.md -or- also to put them in a folder under my username?

KiloQubit commented:
The changes are fine in the directory they're in, because the original author has approved it.  Did you put your own name in the readme as @edwardyeeks has asked?

cbc02009 commented:
> > Since you made these changes and not edwardyeeks this should be filed under your username and not theirs.
> 
> By that do you mean just adding a new row with my username in the root README.md -or- also to put them in a folder under my username?

Sorry, I missed the comment by @edwardyeeks approving adding it to their folder. My mistake.

cbc02009 commented on printer_mods/README.md:
```diff
@@ -100,6 +100,7 @@ like so:
 | |[LED Bar Clip](./eddie/LED_Bar_Clip) |Modular LED Clip for mounting LED Strip to 2020 Extrusions |:x: |:heavy_check_mark: |:heavy_check_mark:|:x:|:heavy_check_mark:|
 |edwardyeeks |[Decontaminator Purge Bucket & Nozzle Scrubber](./edwardyeeks/Decontaminator_Purge_Bucket_&_Nozzle_Scrubber) |Removeable purge bucket with nozzle scrubber |:x: |:heavy_check_mark: |:heavy_check_mark: |:x:|
 | |[V2.4 Z Drive Motor Tensioner Mod](edwardyeeks/V2.4_z_drive_motor_tensioner_mod) |Braced motor tensioner for z drive |:x: |:x: |:heavy_check_mark: |:x:|
+| |[Tridentified V2.4 Power Inlet](edwardyeeks/Tridentified_V2.4_Power_Inlet) |Trident style side power inlet for V2.4 using Z Drive Motor Tensioner Mod |:x: |:x: |:heavy_check_mark: |:x:|
```
Please add the appropriate mark to the trident column

Amarpal89 commented:
Have added myself to the readme table as requested by @edwardyeeks and added correct mark to trident column.

cbc02009 commented:
> Have added myself to the readme table as requested by @edwardyeeks and added correct mark to trident column.

Sorry for the confusion, you don't need to add your own section to the readme since the mod is going into someone else's folder. Can you undo commit 12557b3 ?

Amarpal89 commented:
> > Have added myself to the readme table as requested by @edwardyeeks and added correct mark to trident column.
> 
> Sorry for the confusion, you don't need to add your own section to the readme since the mod is going into someone else's folder. Can you undo commit [12557b3](https://github.com/VoronDesign/VoronUsers/commit/12557b3293c88cf6656ec29924d32b829200b3d0) ?

I've reverted the last commit (and merged upstream changes). Should be all good now.
